### PR TITLE
update processes to allow for coordinated vulnerability disclosure

### PR DESCRIPTION
**Pull request by TravisEz13**
## PR Summary

update processes to allow for coordinated vulnerability disclosure

## PR Checklist

Note: Please mark anything not applicable to this PR `NA`.

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - [x] Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [NA] User facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - [NA] Issue filed - Issue link:
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [NA] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
    - [NA] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.

### CODE REVIEW

File: docs/maintainers/issue-management.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
 # Issue Management
 
+## Security Vulnerabilities
+
+If you believe that there is a security vulnerability in PowerShell Core,
+it **must** be reported to [secure@microsoft.com](https://technet.microsoft.com/en-us/security/ff852094.aspx) to allow for [Coordinated Vulnerability Disclosure](https://technet.microsoft.com/en-us/security/dn467923).
+**Only** file an issue, if secure@microsoft.com has confirmed filing an issue appropriate.
```
Comment (daxian-dbw) [inline docs/maintainers/issue-management.md]: > an issue appropriate

**is**

**Review by daxian-dbw — APPROVED** [review]
LGTM with one comment

**Review by PaulHigin — APPROVED** [review]
LGTM

File: docs/maintainers/issue-management.md [hunk 2]
```diff
@@ -1,8 +1,14 @@
 # Issue Management
 
+## Security Vulnerabilities
+
+If you believe that there is a security vulnerability in PowerShell Core,
+it **must** be reported to [secure@microsoft.com](https://technet.microsoft.com/en-us/security/ff852094.aspx) to allow for [Coordinated Vulnerability Disclosure](https://technet.microsoft.com/en-us/security/dn467923).
+**Only** file an issue, if secure@microsoft.com has confirmed filing an issue is appropriate.
```
Comment (iSazonov) [inline docs/maintainers/issue-management.md]: How can I find out that the approval is received? Should I delete messages that are not able to approve? Maybe the MSFT team would rather exclusively publish such Issues or even announcements?

Comment (TravisEz13) [reply docs/maintainers/issue-management.md]: > How can I find out that the approval is received? 

For someone else, you cannot.  secure@microsoft.com would tell the person reporting the issue if it is appropriate to file an issue.   If this comes up, please alert one of the Microsoft maintainers.  We will report the vulnerability to secure@microsoft.com and take it from there.

> Should I delete messages that are not able to approve?

Don't delete a message yourself. 

> Maybe the MSFT team would rather exclusively publish such Issues or even announcements?

Yes, if it is found to actually be a vulnerability, most likely the filing of the issue will be done or at least coordinated by someone at Microsoft.  The only time I can see that secure@microsoft.com would say that they should file an issue, which is not coordinated with the release of the fix is when the issue is determined not to be a vulnerability.

I don't want to make these instructions too specific until we find a need.  The only need I see right now is if someone does report a vulnerability issue on GitHub, then, we can point to this policy on why we are not responding using GitHub.  Something to the effect of:

> This is not the appropriate place to report this issue, please see Security Vulnerabilities and report this issue to secure@microsoft.com.  I have reported this issue on your behalf in the meantime.

But, we will have to take it on a case by case basis and see what the most appropriate response is.

Comment (iSazonov) [reply docs/maintainers/issue-management.md]: Clear. Thanks!

/cc @markekraus Please read the new policy.

**Pull request merged**